bdk-ffi/src/tx_builder.rs

```diff
@@ -204,6 +206,13 @@ impl TxBuilder {
         })
     }
 
+    pub(crate) fn current_height(&self, height: u32) -> Arc<Self> {
+        Arc::new(TxBuilder {
+            current_height: Some(height),
+            ..self.clone()
+        })
+    }
+
```

Comment:
reviewing commit by commit, this looks good, I wrote a test for it and it passed ✅